src/components/cards/TappableCard.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+
+import * as React from 'react'
+import { TouchableOpacity, View } from 'react-native'
+import FontAwesome5 from 'react-native-vector-icons/FontAwesome5'
+
+import { memo } from '../../types/reactHooks.js'
+import { type Theme, cacheStyles, useTheme } from '../services/ThemeContext.js'
+import { Card } from './Card'
+
+/**
+ * An (optionally) tappable card that displays its children in up to two left/right
+ * sections. If the card is configured to be tappable, a chevron is drawn on the
+ * right side of the card.
+ */
+const TappableCardComponent = ({
+  leftChildren,
+  nonTappable = false,
+  onPress,
+  rightChildren
+}: {
+  leftChildren: React.Node,
+  nonTappable?: boolean,
+  onPress?: any => void | (any => Promise<void>),
```

Comment:
`onPress?: any => void | Promise<void>` would be the same, no?

Reply:
Flow complains

Reply:
`onPress?: any => Promise<void> | void` should work